docnado/docnado.py

```diff
@@ -21,20 +21,20 @@
 import threading
 import traceback
 import subprocess
-import collections
 import platform
-import pathlib
 import requests
+
 from bs4 import BeautifulSoup
+
 from multiprocessing import Pool
+
 from urllib.parse import urlparse
-from collections import defaultdict
 
 from watchdog.observers import Observer
 from watchdog.events import PatternMatchingEventHandler
 
 import timeago
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElementTree
```

Comment:
change to `from xml.etree import ElementTree`

Reply:
Done.